CruzHacks/cruzhacks-2021-website
# Footer - 2nd Iteration

Problem
=======
As a prospective hacker,
I would like to view additional social media links
To become more involved and connected with CruzHacks

[Link to Asana Ticket](https://app.asana.com/0/1197738625416321/1199382769817337/f)

Solution
========
To revise the webpage's `Footer` component, flexbox-based positioning was used lay out new social media links (defined though a mix-in) and specify their responsiveness. All properties are defined and passed in through a `footer.tsx` file and an additional local state variable was defined to specify scrolling. The `Team` button will be implemented following an ongoing pull request using a React router.

Change summary:
---------------
* preliminary footer component draft  a3c6c7e
* fixed icon rendering + responsiveness 08e28b6

Steps to Verify:
----------------
1. Checkout branch using git checkout `feature/footer`
2. Start local development server using `yarn start`
3. Navigate to the homepage to verify that the footer is designed and responds correctly.
